### PR TITLE
🔥 Terminate MSS with extreme prejudice

### PR DESCRIPTION
**Pull request by grdryn**
JIRA: https://issues.jboss.org/browse/AEROGEAR-10077

- Don't install MSS any longer
- Remove MSS from existing clusters

Not removing the CRDs as part of the upgrade, since if people have created CRs for those, then I'd imagine that could break upgrade on that cluster.

## Verification Steps
As the verifier of the PR the following process should be done:

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 

Installation log: https://gist.github.com/grdryn/5e6d60287e5f902a1ae7e6265734230f
RHPDS cluster GUID: `freshmss-4732`

There should be no mobile-security-service namespace

### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

Upgrade log: https://gist.github.com/grdryn/401e6e610c099acc7c0b97fb662cd133
RHPDS cluster GUID: `upgrdryn-af79`

Upgrade should have remove mobile-security-service.

## Is an upgrade task required and are there additional steps needed to test this?

- [x] Yes
- [x] No

Why is Yes/No multiple choice? :grinning:

- [x] Tested Installation
- [x] Tested Uninstallation
- [x] Tested or Created follow on for Upgrade

### CODE REVIEW

File: playbooks/upgrade.yml [hunk 1]
```diff
@@ -32,6 +32,14 @@
...
     with_items: "{{ upgrade_product_roles }}"
 
+  -
+    name: Uninstall Mobile Security Service
+    include_role:
+      name: mobile_security_service
+      tasks_from: uninstall_mss.yml
+    vars:
+      mss_namespace: "{{ eval_mobile_security_service_namespace }}"
+
```
Comment (mikenairn) [inline playbooks/upgrade.yml]: Can revert all the changes in here for the moment. This playbook will only be used after 1.6 is branched. All upgrade work should be done in  playbooks/upgrades/upgrade.yaml

Comment (grdryn) [reply playbooks/upgrade.yml]: Perfect, thanks!

**Comment by grdryn** [comment]
Running an installation & an upgrade on RHPDS clusters now. I forgot to log the fresh install one to a file, so I might not be able to add all of the logs from it (assuming it succeeds).

**Comment by grdryn** [comment]
Both install & upgrade worked successfully (logs included in description).

@mikenairn @maleck13 it probably makes more sense for one of you from RHMI to review this than someone from my team, wdyt?

**Comment by mikenairn** [comment]
/lgtm

It all looks fine to me, but i haven't tested it, are we happy that it's been tested enough?

We will also need a PR to remove the DeLorean jobs for mss and mss-operator https://github.com/integr8ly/ci-cd/tree/master/jobs/delorean

**Comment by grdryn** [comment]
> We will also need a PR to remove the DeLorean jobs for mss and mss-operator https://github.com/integr8ly/ci-cd/tree/master/jobs/delorean

I've created the following PR there just now: https://github.com/integr8ly/ci-cd/pull/356

**Pull request merged**